service/policy/db/migrations/20250407000000_add_non_data_resources.sql

```diff
@@ -0,0 +1,51 @@
+-- +goose Up
+-- +goose StatementBegin
+
+CREATE TABLE IF NOT EXISTS non_data_resource_groups (
```

Comment:
We have `groups:values` here, and some other ideas for the parent are `class`, `type`, `definition` (which is used in attributes but could be confusing with no rule), `category` (which is used for entity types). Any of those sound better to you?

Reply:
What if we just went with:

`non_data_resources`  and `non_data_resource_values`

As that would be almost consistent with `attribute_definitions` and `attribute_values` and gets rid of the annoying `groups` suffix.

Reply:
I like non_data_resources and non_data_resource_values

Reply:
@ttschampel what are your thoughts on `registered_resources` instead of `non_data_resources` to describe the objects by what they are instead of by what they are not?

Reply:
I'm good with this